packages/circuit-ui/components/ListItemGroup/ListItemGroup.tsx

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ReactElement, forwardRef, Ref, HTMLProps, ReactNode } from 'react';
+import { css } from '@emotion/core';
+
+import styled, { StyleProps } from '../../styles/styled';
+import { ReturnType } from '../../types/return-type';
+import { ListItemProps } from '../ListItem';
+
+interface BaseProps {
+  /**
+   * Children of type ListItem.
+   */
+  children: ReactElement<ListItemProps>[] | ReactElement<ListItemProps>;
+  /**
+   * Display a main label for this group of list items.
+   */
+  title?: ReactNode;
+  /**
+   * Display a secondary label for this group of list items.
+   */
+  suffix?: ReactNode;
+  /**
+   * Make the list item group suitable for full-width presentation on smaller resolutions.
+   */
+  immersive?: boolean;
```

Comment:
For other components we've used the pattern `variant: "immersive" | ...` rather than a boolean. Do you think we could also apply it here? Not sure what the other variant name would be (I like immersive though)

Reply:
Yeah, I know that's how we use it in the Modal. The problem is that the `ListItem` already has the `action` and `navigation` variants and they should be able to be used in conjunction with `immersive`.
I think that in Figma they call that **presentation** and they have `plain` (immersive), `inset` (with borders) and `borderless` (without borders) as options, though I was told to disregard `borderless` for the time being. In all honesty I like `immersive` much better than `plain` in terms of wording.